docs/infrastructure/eks-quickstart.md

```diff
@@ -1,204 +1,159 @@
 
-```bash
-kubectl get pods -A
-```
+    ```bash
+    kubectl get pods -A
+    ```
 
-you'll see that nothing is scheduled. That's because you haven't yet allowed
-the worker nodes to join the cluster
-(see the [EKS docs](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html) for more info). We'll do that on step 3.
+    You'll notice that nothing is scheduled. That's because we haven't yet allowed the worker nodes to join the cluster (see the [EKS docs](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html) for more details). We'll do that on the next section.
 
-### 3. `setup` - configure your EKS cluster to deploy codacy
+## 4. Set up the cluster to run Codacy
 
-Some additional setup  is necessary to run Codacy on the cluster you just created.
-For one, you need to allow workers to join the cluster. Docker pull secrets need to be added to the `codacy` namespace, which will also be added here,
-and the following helm charts will be installed:
-[kubernetes-dashboard](https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/),
-[nginx-ingress](https://github.com/helm/charts/tree/master/stable/nginx-ingress), and
-[cert-manager](https://github.com/jetstack/cert-manager).
+Some additional setup is necessary to run Codacy on the newly created cluster, such as allowing workers to join the cluster and installing the following Helm charts:
 
-To do it run
+* [Kubernetes Dashboard](https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/)
+* [nginx-ingress](https://github.com/helm/charts/tree/master/stable/nginx-ingress)
+* [cert-manager](https://github.com/jetstack/cert-manager)
 
-```bash
-terraform init && terraform apply
-```
+Set up the cluster to run Codacy:
 
-You'll be prompted to input Codacy's docker hub repo password.
+1. Initialize Terraform and apply the changes inside the `setup/` directory, then follow Terraform's instructions:
 
-If you'd like to connect to the kubernetes dashboard, get the admin token
-by running
+    ```bash
+    cd ../setup/
+    terraform init && terraform apply
+    ```
 
-```bash
-terraform output admin_token
-```
+1. To connect to the Kubernetes Dashboard, run:
 
-and copy the outputed value. To connect to the dashboard run
+    ```bash
+    kubectl proxy
+    ```
 
-```bash
-kubectl proxy
-```
+    Open the following URL on a browser and select `token`:
 
-and then connect to the dashboard URL
+    <http://127.0.0.1:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:https/proxy>
 
-<http://127.0.0.1:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:https/proxy>
+    Run the following command to obtain the admin token required to connect to the Kubernetes Dashboard:
 
-select `token` and paste the value you saved above.
+    ```bash
+    terraform output admin_token
+    ```
 
-## Uninstalling
+## Uninstalling the Amazon EKS cluster
```

Comment:
We could think of a better place to have these instructions later, but for now we can leave them here.